src/index.js

```diff
@@ -64,7 +66,28 @@ const display = (taskObj) => {
   divMenu2.addEventListener('click', () => {
     deleteTask(liTask, tasks);
   });
+
+  if (taskObj.completed === true) {
+    descriptionTask.classList.add('check');
+    check.checked = true;
+  }
+
+  check.addEventListener('click', () => {
+    if (check.checked) {
+      descriptionTask.classList.add('check');
+      taskObj.completed = true;
+      localStorage.setItem('taskList', JSON.stringify(tasks));
+    } else {
+      descriptionTask.classList.remove('check');
+      taskObj.completed = false;
+      localStorage.setItem('taskList', JSON.stringify(tasks));
+    }
```

Comment:
- [ ] As per requirements, the event listener on the checkbox should be (`change`).
- [ ] I would also recommend creating one new JavaScript file and import as a module:
   - it should contain methods related to the status updates (`completed`: `true` / `false`).